jsoup/src/main/java/org/jsoup/parser/Token.java

```diff
@@ -443,9 +443,9 @@ public String toString() {
     }
 
     final boolean isDoctype() {
-            if(type == TokenType.Doctype)
-                    return true;
-            return false;
+        if(type == TokenType.Doctype)
+                return true;
```

Comment:
Needs to fix indentation to match formatting 8 spaces -> 4 spaces

Reply:
@ahmedfahad04 please address this in the next 2-3 days, or we'll assign to someone else.

Reply:
Fixed the indentation ✅